src/web/app/pages-admin/admin-notifications-page/notification-edit-form/notification-edit-form.component.scss

```diff
@@ -0,0 +1,37 @@
+.card-plain {
+  background-color: #EAEFF5;
+  background-image: none;
+  border: 0;
+  box-shadow: 0 0 0 0;
+}
+
+.close-header {
+  padding-top: .75rem;
+  padding-right: 1.25rem;
+  margin-bottom: 0;
+}
+
+.font-bold {
```

Comment:
There's a `font-weight-bold` class already available in bootstrap.

Reply:
Thanks for the comment. It has been removed and places where it was used are replaced with font-weight-bold class.